packages/web-components/src/components/lightbox-media-viewer/__stories__/README.stories.mdx

```diff
@@ -52,6 +52,20 @@ document.getElementById('my-video').open = true;
 <dds-lightbox-video-player-container id="my-video" video-id="1_9h94wo6b"> </dds-lightbox-video-player-container>
 ```
 
+## Video player with onClick Lightbox Trigger
+
+### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/lightbox-media-viewer/lightbox-video-player-container.js';
+```
+
+### HTML
+
+```html
+<dds-lightbox-video-player-container render-thumbnail player-mode="thumbnail" id="my-video" video-id="1_9h94wo6b"> </dds-lightbox-video-player-container>
+```
+
```

Comment:
Hey @proeung , saw some back and forth on where the story should be, is this still in lightbox media viewer or needs to move to the video player README?

Reply:
@jeffchew Since we're using the `<dds-lightbox-video-player-container>` component to get the Lightbox overlay functionality, I think it's important to note this within the README of the `lightbox-media-viewer`, however, I did note this doc within the README of the `video-player` as part of this commit that was added this morning. https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/5581/commits/a334359c0b6682880efc314021f2d6a0eae7acf8#diff-df79197e91cdbc818e3db1cdd74df7aa70f7d78280af8a2f73a6a9ff10efde4aR49